core/src/i18n/en.yaml

```
@@ -742,8 +757,16 @@ gateways:
         credentialName: Credential Name
         mode: TLS Mode
         tls: TLS
-    messages:
-      success: Gateway created.
+        ca-certificates: CA Certificate
+        http-redirect: HTTP Redirect
+        disabled-for-non-https: TLS settings are not available for a non-HTTPS protocol.
+        http-redirect-description: Enable 301 redirect for all HTTP connections.
+        private-key: Private Key
+        server-certificate: Server Certificate
+        max-protocol-version: Max Protocol Version
+        min-protocol-version: Min Protocol Version
+        messages:
+          invalid-tls-warning: TLS Server of mode SIMPLE or MUTUAL require either credential name or private key and server certificate pair.
```

```suggestion
          invalid-tls-warning: TLS Server of mode SIMPLE or MUTUAL need either credential name or private key and server certificate pair.
```
Ambiguous, this could mean the following
- either a "credential name" or the "private key + server certificate" pair
- either the "credential name + server certificate pair" or the "private key + server certificate" pair
- several other combinations of the words :D 
 
 We need to add commas etc to clarify it.

---

Done in `Update translations`.

---

(Hopefully)

---

I think so! :)